src/Model_Hydro/CPU_Hydro/CPU_Shared_DualEnergy.cpp

```diff
@@ -153,7 +153,8 @@ real Hydro_Con2Dual( const real Dens, const real MomX, const real MomY, const re
 // calculate pressure and convert it to the dual-energy variable
 // --> note that DE_ENPY only works with EOS_GAMMA, which does not involve passive scalars
    Pres = Hydro_Con2Pres( Dens, MomX, MomY, MomZ, Engy, NULL, CheckMinPres_No, NULL_REAL, Emag,
-                          EoS_DensEint2Pres, EoS_AuxArray_Flt, EoS_AuxArray_Int, EoS_Table, NULL );
+                          EoS_DensEint2Pres, EoS_GuessHTilde, EoS_HTilde2Temp,
```

Comment:
"EoS_GuessHTilde" and "EoS_HTilde2Temp" are undefined when compiling with `DUAL_ENERGY`  enabled (`--model=HYDRO --dual=DE_ENPY`)